docker/develop/Dockerfile

```diff
@@ -185,5 +185,13 @@ RUN git clone git://sourceware.org/git/valgrind.git /tmp/valgrind; \
     make -C /tmp/valgrind install; \
     rm -rf /tmp/valgrind
 
+# install ed25519
+RUN git clone git://github.com/warchant/ed25519.git /tmp/ed25519; \
+    (cd /tmp/ed25519 ; git checkout 0d31b082e0f0d4144fe598fb0f2a120e36de2a8f); \
```

Comment:
Version does not correspond to the one in Finded25519.cmake

Reply:
Fixed.